templates/CRM/Event/Form/Selector.tpl

```diff
@@ -55,9 +55,9 @@
         {/foreach}
         </td>
     {else}
-        <td class="crm-participant-participant_fee_level">{if !$row.paid && !$row.participant_fee_level} {ts}(no fee){/ts}{else} {$row.participant_fee_level}{/if}</td>
+        <td class="crm-participant-participant_fee_level">{if !$row.paid && (!array_key_exists('participant_fee_level', $row) || !$row.participant_fee_level)} {ts}(no fee){/ts}{elseif array_key_exists('participant_fee_amount', $row)}{$row.participant_fee_level}{/if}</td>
```

Comment:
Should this be level instead of amount?

Reply:
short version = yes.
long version = essay about how my brain is melting down & I had to read that about 25 times to be sure